app/src/Level2_simple_agentic_with_websearch.ipynb

```diff
@@ -0,0 +1,220 @@
+      "\u001b[0m\u001b[33m\n",
+      "\u001b[0m\u001b[33m6\u001b[0m\u001b[33m.\u001b[0m\u001b[33m **\u001b[0m\u001b[33mRed\u001b[0m\u001b[33m Hat\u001b[0m\u001b[33m Open\u001b[0m\u001b[33mShift\u001b[0m\u001b[33m AI\u001b[0m\u001b[33m**:\u001b[0m\u001b[33m There\u001b[0m\u001b[33m's\u001b[0m\u001b[33m a\u001b[0m\u001b[33m technology\u001b[0m\u001b[33m preview\u001b[0m\u001b[33m of\u001b[0m\u001b[33m Red\u001b[0m\u001b[33m Hat\u001b[0m\u001b[33m Open\u001b[0m\u001b[33mShift\u001b[0m\u001b[33m L\u001b[0m\u001b[33mights\u001b[0m\u001b[33mpeed\u001b[0m\u001b[33m,\u001b[0m\u001b[33m which\u001b[0m\u001b[33m now\u001b[0m\u001b[33m adds\u001b[0m\u001b[33m support\u001b[0m\u001b[33m for\u001b[0m\u001b[33m Red\u001b[0m\u001b[33m Hat\u001b[0m\u001b[33m Open\u001b[0m\u001b[33mShift\u001b[0m\u001b[33m AI\u001b[0m\u001b[33m and\u001b[0m\u001b[33m Red\u001b[0m\u001b[33m Hat\u001b[0m\u001b[33m Enterprise\u001b[0m\u001b[33m Linux\u001b[0m\u001b[33m AI\u001b[0m\u001b[33m as\u001b[0m\u001b[33m model\u001b[0m\u001b[33m providers\u001b[0m\u001b[33m,\u001b[0m\u001b[33m enabling\u001b[0m\u001b[33m organizations\u001b[0m\u001b[33m to\u001b[0m\u001b[33m use\u001b[0m\u001b[33m Red\u001b[0m\u001b[33m Hat\u001b[0m\u001b[33m as\u001b[0m\u001b[33m the\u001b[0m\u001b[33m source\u001b[0m\u001b[33m of\u001b[0m\u001b[33m the\u001b[0m\u001b[33m underlying\u001b[0m\u001b[33m LL\u001b[0m\u001b[33mM\u001b[0m\u001b[33m.\u001b[0m\u001b[33m\n",
+      "\u001b[0m\u001b[33m\n",
+      "\u001b[0m\u001b[33mRemember\u001b[0m\u001b[33m to\u001b[0m\u001b[33m check\u001b[0m\u001b[33m the\u001b[0m\u001b[33m release\u001b[0m\u001b[33m notes\u001b[0m\u001b[33m for\u001b[0m\u001b[33m a\u001b[0m\u001b[33m comprehensive\u001b[0m\u001b[33m list\u001b[0m\u001b[33m of\u001b[0m\u001b[33m updates\u001b[0m\u001b[33m.\u001b[0m\u001b[97m\u001b[0m\n",
+      "\u001b[30m\u001b[0m"
+     ]
+    }
+   ],
+   "source": [
+    "session_id = agent.create_session(\"test-session\")\n",
+    "for prompt in user_prompts:\n",
+    "    cprint(f\"User> {prompt}\", \"green\")\n",
+    "    response = agent.create_turn(\n",
+    "        messages=[\n",
+    "            {\n",
+    "                \"role\": \"user\",\n",
+    "                \"content\": prompt,\n",
+    "            }\n",
+    "        ],\n",
+    "        session_id=session_id,\n",
+    "    )\n",
+    "    for log in EventLogger().log(response):\n",
+    "        log.print()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Key Takeaways\n",
+    "- This tutorial demonstrates how to set up and use a single tool with Agentic.\n",
+    "- It highlights the flexibility of switching between remote and local environments.\n",
```

Comment:
It is not super clear to me that this was shown in this notebook. Can you just add some more text that makes it clear your able to run this same notebook against any correctly configured llama stack server?